src/instructlab/model/backends/backends.py

```diff
@@ -86,6 +88,37 @@ def shutdown(self):
         """Shutdown serving backend"""
 
 
+def is_model_safetensors(model_path: pathlib.Path) -> bool:
+    """
+    Check if provided path to model represents directory containing a safetensors representation
+    of a model. Directory must contain a specific set of files to qualify as a safetensors model directory
+    Args:
+        model_path (Path): The path to the model directory
+    Returns:
+        bool: True if the model is a safetensors model, False otherwise.
+    """
+
+    if not model_path.is_dir():
+        return False
```

Comment:
At this point we can't tell whether there is an issue with the content of the directory or if model_path was not a directory at all.

Reply:
removed, thanks!